src/FSharpPlus/Extensions/Validators.fs

```diff
@@ -0,0 +1,75 @@
+﻿namespace FSharpPlus
+
+[<RequireQualifiedAccess>]
+module RequiredValidation =
+    open System
+    open FSharpPlus.Data
+    
+    let inline validate error f v =
+        if f v then
+            Success v
+        else
+            Failure [ error ]
+
+    let inline string error value =
```

Comment:
I'm unsure if just calling it `string` makes it obvious. Would it be better to call it `notNullOrWhiteSpace` ?